components/user-mgt/org.wso2.carbon.identity.user.profile/src/main/java/org/wso2/carbon/identity/user/profile/mgt/UserProfileAdmin.java

```diff
@@ -463,6 +474,64 @@ public UserProfileDTO getUserProfile(String username, String profileName)
         return profile;
     }
 
+
+    private int getDomainID(String domain, int tenantId) throws UserProfileException {
+
+        int domainId = 0;
+
+
+
+
+        String domainIdQuery = "SELECT UM_DOMAIN_ID FROM UM_DOMAIN WHERE UM_DOMAIN_NAME=? AND UM_TENANT_ID=?";
+
+        try {
+
+            Connection indbs = JDBCPersistenceManager.getInstance().getDBConnection();
+
+            PreparedStatement domainIdStatement = indbs.prepareStatement(domainIdQuery);
+
+            domainIdStatement.setString(1, domain);
+            domainIdStatement.setInt(2, tenantId);
+
+            ResultSet resultSet = domainIdStatement.executeQuery();
+
+            while (resultSet.next()) {
+                domainId = resultSet.getInt("UM_DOMAIN_ID");
```

Comment:
ran SONAR. Will fix the thing.. :D

On Mon, Jun 15, 2015 at 12:54 PM, Prasad Tissera <notifications@github.com>
wrote:

> In
> components/user-mgt/org.wso2.carbon.identity.user.profile/src/main/java/org/wso2/carbon/identity/user/profile/mgt/UserProfileAdmin.java
> <https://github.com/wso2/carbon-identity/pull/435#discussion_r32396334>:
>
> > +
> > +        String domainIdQuery = "SELECT UM_DOMAIN_ID FROM UM_DOMAIN WHERE UM_DOMAIN_NAME=? AND UM_TENANT_ID=?";
> > +
> > +        try {
> > +
> > +            Connection indbs = JDBCPersistenceManager.getInstance().getDBConnection();
> > +
> > +            PreparedStatement domainIdStatement = indbs.prepareStatement(domainIdQuery);
> > +
> > +            domainIdStatement.setString(1, domain);
> > +            domainIdStatement.setInt(2, tenantId);
> > +
> > +            ResultSet resultSet = domainIdStatement.executeQuery();
> > +
> > +            while (resultSet.next()) {
> > +                domainId = resultSet.getInt("UM_DOMAIN_ID");
>
> Did you run sonar / Findbugs? We need to fix all the issues in new code.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/wso2/carbon-identity/pull/435/files#r32396334>.
>



-- 
-Damith -